libs/libhtp/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2019-2020 CZ.NIC z.s.p.o. (http://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libhtp
+PKG_VERSION:=0.5.37
+PKG_RELEASE=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/OISF/libhtp/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=0f8296ce9db815bbedc1c8a0b738dd25146fdaf910995a8c5f8e147799902b7e
+
+PKG_INSTALL:=1
+PKG_FIXUP:=autoreconf
+
+PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
```

Comment:
Was this copied from the turris repo?

Reply:
It has the Copyright Block still, but yes, I believe so.  Although the version I pulled was outdated and had to be changed.

Reply:
It depends, which branch you used, though. I updated it a week ago:
https://gitlab.nic.cz/turris/turris-os-packages/-/commit/90a75aae93f39f65de26edb4a5a074fbb413305c

Reply:
@BKPepe Was the one you linked to in the Suricata6 PR.  Was for version 0.5.33 I believe.  I dropped the -fno-stack-protector and repetitious Build/Compile from that version, and the ICONV stuff simply for expediency because I was confused over the iconv-full vs iconv/host and was getting errors.

Reply:
You should assign to yourself if you want to be maintainer, because currently I don't want to take maintainership of this package as there is no Suricata present in this repository. Also, I didn't use try it on OpenWrt master.

Which errors did you get?